SPECS/nexus-oss.spec

```diff
@@ -109,14 +108,22 @@ rm -rf $RPM_BUILD_ROOT
 %files
 %defattr(-,root,root,-)
 %doc
-/etc/init.d/%{name}
 %attr(-,%{name},%{name}) /etc/%{name}
 %attr(-,%{name},%{name}) /var/lib/%{name}
 %attr(-,%{name},%{name}) /var/log/%{name}
 %attr(-,%{name},%{name}) /var/run/%{name}
 %attr(-,%{name},%{name}) /usr/share/%{name}
+%if %{use_systemd}
+%{_unitdir}/%{name}.service
+%else
+/etc/init.d/%{name}
+%endif
 
 %changelog
+* Sat Dec 30 2017 Anton Patsev <patsev.anton@gmail.com> - 3.6.2.01-2
```

Comment:
The version was incorrect, it had to be 2.14.5.02-2, but I fixed it at master.

Reply:
I am sorry. I copied and didn`t change.